tests/bluetooth/tester/default.conf

```diff
@@ -9,5 +9,17 @@ CONFIG_BT_ATT_PREPARE_COUNT=4
 CONFIG_BT_GATT_CLIENT=y
 CONFIG_BT_L2CAP_DYNAMIC_CHANNEL=y
 CONFIG_BT_PRIVACY=y
+
+CONFIG_BT_MESH=y
+CONFIG_BT_MESH_PROV=y
+CONFIG_BT_MESH_RELAY=y
+CONFIG_BT_MESH_PB_ADV=y
+CONFIG_BT_MESH_PB_GATT=y
+CONFIG_BT_MESH_GATT_PROXY=y
+CONFIG_BT_MESH_SUBNET_COUNT=5
```

Comment:
I don't think any of the Mesh tests requires 5 subnets. 2 should be enough.

Reply:
ok